plugins/amd-gpu/meson.build

```diff
@@ -1,4 +1,7 @@
-if host_machine.system() == 'linux' and libdrm_amdgpu.found()
+if host_machine.system() != 'linux' or not libdrm_amdgpu.found()
+  subdir_done()
+endif
```

Comment:
can we use subdir_done multiple times? e.g. this might be a lot clearner if it was:

	if host_machine.system() != 'linux'
	  subdir_done()
	endif
	if not libdrm_amdgpu.found()
	  subdir_done()
	endif

... if that works I'm happy to help.

Reply:
Yeah that should work.